macros/src/main/scala/spire/macros/Syntax.scala

```diff
@@ -42,7 +42,7 @@ object Ops extends machinist.Ops {
       (uesc('⊼'), "nand"),
       (uesc('⊽'), "nor"))
 
-  def unopWithEv2[Ev1, R](c: Context)(ev1: c.Expr[Ev1]): c.Expr[R] = {
+  override def unopWithEv2[Ev1, R](c: Context)(ev1: c.Expr[Ev1]): c.Expr[R] = {
```

Comment:
We can remove this method now that we are using a more recent version of Machinist.